README.md

```diff
@@ -23,14 +23,35 @@ up and running to develop and test changes. Follow these steps:
 
 # Build the development image
 
+Clone the repo locally:
+
 ```bash
 git clone https://github.com/talis/talis-php.git
 cd talis-php
-ant build
```

Comment:
`ant build` wouldn't work for me. It requires Java which I don't have installed. Hence, the update to the instructions in order to do a build manually. Should I leave a note in about using `ant`?

Reply:
No I think its good to remove it.